src/pages/resultsView/mutation/Mutations.tsx

```diff
@@ -238,6 +242,33 @@ export default class Mutations extends React.Component<
         }
     }
 
+    @autobind
+    @action
+    protected getDownloadData(dataType?: DataType): string {
+        var flatdata: any = '';
```

Comment:
```suggestion
        let flatdata: string = '';
```

Reply:
yes, lets always use const or let

Reply:
and ts will infer the type, so  `const flatdata = ''`  is sufficient